patches/api/0003-Test-changes.patch

```diff
@@ -187,20 +187,31 @@ index 64e7aef6220097edefdff3b98a771b988365930d..7ff939ea41417bad3a436a87c89d5efa
      }
  
      private static void collectClasses(@NotNull File from, @NotNull Map<String, ClassNode> to) throws IOException {
-@@ -125,6 +186,12 @@ public class AnnotationTest {
+@@ -125,6 +186,23 @@ public class AnnotationTest {
          }
      }
  
 +    // Paper start - skip class if it's @NullMarked
-+    private static boolean isClassNullMarked(@NotNull ClassNode clazz) {
++    private static boolean isClassNullMarked(@NotNull ClassNode clazz, @NotNull Map<String, ClassNode> allClasses) {
++        if (clazz.nestHostClass != null) {
```

Comment:
While technically useless nitpick, this would fail if the annotation was present on the inner class but not on the outer one?

E.g. this may instead have to be something like
```java
private static boolean isClassNullMarked(@NotNull ClassNode clazz, @NotNull Map<String, ClassNode> allClasses) {
    if (isClassNullMarked0(clazz)) return true;

    String parentClass = clazz.nestHostClass;
    while (parentClass != null) {
        final ClassNode parentClassNode = allClasses.get(parentClass);
        if (parentClassNode == null) return false;

        if (isClassNullMarked0(parentClassNode)) return true;

        parentClass = parentClassNode.nestHostClass;
    }
    return false;
}
```
if we truely care about supporting inner types correctly in this. Not that that is really important lol